packages/api/src/command/medical/patient/check-brief-enabled.ts

```diff
@@ -0,0 +1,27 @@
+import { capture, out } from "@metriport/core/util";
+import BadRequestError from "../../../errors/bad-request";
+import { isAiBriefEnabledForCx } from "../../../external/aws/app-config";
+
+export async function checkAiBriefEnabled({
+  cxId,
+  generateAiBrief,
+}: {
+  cxId: string;
+  generateAiBrief: boolean;
+}) {
+  const { log } = out(`AI Brief for cxId: ${cxId}`);
+  const isAiBriefFeatureFlagEnabled = await isAiBriefEnabledForCx(cxId);
+  if (!isAiBriefFeatureFlagEnabled && generateAiBrief) {
+    const msg = `CX requires AI Brief feature`;
+    log(msg);
+    capture.message(msg, {
+      extra: {
+        cxId,
+        generateAiBrief,
+        isAiBriefFeatureFlagEnabled,
+      },
+      level: "warning",
```

Comment:
This could prob be `info`, no?

Reply:
I was debating between the two, and thought warning could be more appropriate in that it would draw more attention on our end. But conceptually, info is more fitting